src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using Microsoft.PowerFx.Core.Types.Enums;
+
+namespace Microsoft.PowerFx.Core.Public.Config
```

Comment:
>Microsoft.PowerFx.Core.Public.Config [](http://example.com/codeflow?start=10&length=36)

nit: For public namespaces, can we stick with just "Microsoft.PowerFx". 


(@lesaltzm ? )

Reply:
All the types are located in Microsoft.PowerFx.Core.Public.Types, Values are in the Value namespace.

Reply:
Mike has a good point though, our file organization shouldn't be part of the public contract, I got that wrong when doing the refactor. I'd be ok if we moved Types/Values to just be Microsoft.PowerFx.Core (or .Types/.Values if we'd prefer putting more meaning into the namespace)

Reply:
So do we want to rename all the namespaces in the Public folder in this PR?

Reply:
In a subsequent PR, seems outside the scope of this one. For this one, I'd just add this class as Microsoft.PowerFx.Core

Reply:
Changed the namespace